code/modules/mob/living/silicon/robot/robot_vr.dm

```diff
@@ -30,3 +30,42 @@
 			overlays += "[module_sprites[icontype]]-sleeper_g"
 		if(sleeper_r == 1)
 			overlays += "[module_sprites[icontype]]-sleeper_r"
+
+/mob/living/silicon/robot/Move(a, b, flag)
+
+	. = ..()
+
+	if(module)
+		if(module.type == /obj/item/weapon/robot_module/scrubpup)
+			var/turf/tile = loc
+			if(isturf(tile))
```

Comment:
Sounds pretty good. I pretty much just copied this whole part from the normal janiborg.